cli/src/base/AppCommandBase.ts

```diff
@@ -27,12 +31,18 @@ export default abstract class AppCommandBase extends MembershipsCommandBase {
         return prev
       }
       const promptValue = await this.simplePrompt<string>({ message: this.appMetadataForm[key as keyof IAppMetadata] })
+      if (promptValue === '') {
+        return prev
+      }
       prev[key] = this.processValue(key, promptValue)
       return prev
     }, Promise.resolve({} as Record<string, string | string[]>))
   }
 
   processValue(key: string, value: string): string | string[] {
+    if (value === '-') {
+      return ''
+    }
```

Comment:
Nice solution!
The only field it doesn't work for is `platforms`, but arrays are tricky in protobuf in general, because there is no difference between empty array and no value provided (in terms of encoding) so best we can do in this case is use `['']` (array with empty string)